### PR TITLE
sdk: support window

### PR DESCRIPTION
**Pull request by JeanTracker**
NUGU SDK supports window using msys2.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -250,10 +265,16 @@ IF(VENDOR_PKGCONFIG)
...
+	SET(GIO_PKGCONFIG gio-windows-2.0)
+ELSE()
+	SET(GIO_PKGCONFIG gio-unix-2.0)
+ENDIF()
+
 IF(BUILD_LIBRARY)
 	# Set NUGU SDK library default dependency
 	SET(DEFAULT_LIB_DEPENDENCY
-		glib-2.0 gio-2.0 gio-unix-2.0 gthread-2.0
+		glib-2.0 gio-2.0 gthread-2.0 openssl ${GIO_PKGCONFIG}
```
Comment (webispy) [inline CMakeLists.txt:277]: `openssl` moved to `SET(CURL_PKGCONFIG zlib openssl)`
https://github.com/nugu-developers/nugu-linux/pull/938

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -23,11 +23,14 @@ ELSE()
...
-	${CURL_LIBRARY} njson
+	${CURL_LIBRARY} njson -lstdc++
 	${pkgs_LDFLAGS}
 	${builtin_plugin_pkgs_LDFLAGS}
 	${BUILTIN_PLUGIN_LIBS}
-	${LDFLAG_SOCKET} ${LDFLAG_DL})
+	${LDFLAG_SOCKET}
+	${LDFLAG_WINSOCK}
+	${LDFLAG_DL}
+	${pkgs_LDFLAGS})
```
Comment (webispy) [inline src/CMakeLists.txt:33]: `${pkgs_LDFLAGS}` duplicated (Line 27)

Comment (JeanTracker) [reply src/CMakeLists.txt:33]: I fixed it.

File: src/base/nugu_winsock.c [hunk 3]
```diff
@@ -0,0 +1,233 @@
+#ifdef __MSYS__
```
Comment (webispy) [inline src/base/nugu_winsock.c:1]: Please add license boilerplates

**Review by webispy — APPROVED** [review]

**Pull request merged**